cityofaustin/atd-moped
# Ignore phase date confirmed activities in activity log

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14351

## Testing
**URL to test:** 
https://deploy-preview-1181--atd-moped-main.netlify.app/moped/projects/1921?tab=activity_log
**Steps to test:**

Edit a phase's start or end date. Set the date to be in the past. The activity log should not reflect that the confirmed date was changed. (Currently in staging, if you set a date to be in the past, you get "Edited the phase Scoping by updating the start date, " and a hanging comma. 

---
#### Ship list
